src/deprecated.jl

```diff
@@ -0,0 +1,3 @@
+export current_device
+
+Base.@deprecate current_device() device()
```

Comment:
```suggestion
@deprecate current_device() device()
```

I think it's only `@deprecate_binding` that isn't exported.